CHANGELOG.md

```diff
@@ -1,44 +1,31 @@
 # nf-core/coproid: Changelog
 
-## v.1.1.1
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-### Fixed
+## v2.0dev - [date]
```

Comment:
Can you remove the `dev` here?

Reply:
versions should also follow the semantic versioning, so `2.0.0` is the correct version here